lightning-background-processor/src/lib.rs

```diff
@@ -143,6 +145,7 @@ impl BackgroundProcessor {
 			log_trace!(logger, "Calling ChannelManager's timer_tick_occurred on startup");
 			channel_manager.timer_tick_occurred();
 
+			let event_handler = NetworkUpdateHandler::new(network_graph, &*logger, event_handler);
```

Comment:
Could we update the [top-level BackgroundProcessor docs](https://github.com/rust-bitcoin/rust-lightning/blob/main/lightning-background-processor/src/lib.rs#L28) for this new behavior?